doc/manual/generate-builtins.nix

```
@@ -1,24 +1,30 @@
 let
-  inherit (builtins) concatStringsSep attrNames;
+  inherit (builtins) concatStringsSep attrValues mapAttrs;
+  inherit (import ./utils.nix) optionalString squash;
 in
 
 builtinsInfo:
 let
-  showBuiltin = name:
+  showBuiltin = name: { doc, args, arity, experimental-feature }:
     let
-      inherit (builtinsInfo.${name}) doc args;
+      xpNotice = optionalString (experimental-feature != null) ''
```

```suggestion
      experimentalNotice = optionalString (experimental-feature != null) ''
```

All this is internal, but self-descriptive names are a key feature of developer experience.